config/formats.js

```diff
@@ -319,8 +319,8 @@ exports.Formats = [
 		],
 
 		mod: 'dancerability',
-		ruleset: ['OU'],
 		searchShow: false,
+		ruleset: ['[Gen 7] OU'],
```

Comment:
@urkerab goes and finds other formats where he's forgotten the `[Gen 7]` in the ruleset...